docs/source/builtin_types.rst

```diff
@@ -31,10 +31,10 @@ strings and ``Dict[Any, Any]`` is a dictionary of dynamically typed
 (arbitrary) values and keys. ``List`` is another generic class. ``Dict`` and
 ``List`` are aliases for the built-ins ``dict`` and ``list``, respectively.
 
-``Iterable``, ``Sequence``, and ``Mapping`` are generic types that
-correspond to Python protocols. For example, a ``str`` object or a
-``List[str]`` object is valid
-when ``Iterable[str]`` or ``Sequence[str]`` is expected. Note that even though
-they are similar to abstract base classes defined in :py:mod:`collections.abc`
-(formerly ``collections``), they are not identical, since the built-in
-collection type objects do not support indexing.
+``Iterable``, ``Sequence``, and ``Mapping`` are generic types that correspond to
```

Comment:
Is this just realigning the lines? Not sure that's useful because the rendered output is the same and it makes the history harder to follow.

Reply:
I rephrase to mention Python 3.9, but yeah the top is just realignment. I'll make my editor do less next time.